src/reporting/tezos/TezosConseilClient.ts

```diff
@@ -48,10 +48,11 @@ export namespace TezosConseilClient {
      * @param network Tezos network to query, mainnet, alphanet, etc.
      * @param hash Block hash to query for.
      */
-    export async function getBlock(serverInfo: ConseilServerInfo, network: string, hash: string): Promise<any[]> {
+    export async function getBlock(serverInfo: ConseilServerInfo, network: string, hash: string): Promise<any> {
         const query = ConseilQueryBuilder.setLimit(ConseilQueryBuilder.addPredicate(ConseilQueryBuilder.blankQuery(), 'hash', ConseilOperator.EQ, [hash], false), 1);
 
-        return getTezosEntityData(serverInfo, network, BLOCKS, query);
+        const r = await getTezosEntityData(serverInfo, network, BLOCKS, query);
+        return r[0];
```

Comment:
We should validate that at least one result was returned, else return an explicit error.